src/models/Activity.js

```diff
@@ -24,5 +25,16 @@ ActivitySchema.virtual('activityHistories', {
   sort: { date: -1 },
 });
 
+// eslint-disable-next-line consistent-return
+function setQuizCount() {
+  if (this.cards.length && this.cards[0].template) {
```

Comment:
pourquoi le test de `this.cards[0].template` ?

Reply:
Enfait ca me permet de verifier que le template est present et que ce ne sont pas que des id. Si ce ne sont que des ids, cela signifie que l'on ne souhaite pas avoir le quizCount et donc je ne renvoie rien (c'est pour ca que j'avais mis le disable consistent-return). 
Sinon le risque c'est d'avoir des quizCount a 0 alors qu'il y a bel et bien des quiz dans l'activité.

Reply:
tu ne peux pas `return; ` ou un `return null; ` ?

Reply:
`return;` ne permet pas d'enlever le `// eslint-disable-next-line consistent-return` et en ajoute un `/* eslint-disable no-useless-return */`.
`return null;` envoie du coup quizCount: null au mobile alors que ce n'est pas utile (et potentiellement faux).